.gitignore

```diff
@@ -1 +1,2 @@
-bmc-test-go
+bmc-test-cross
+bmc-test-native
```

Comment:
Call me nitpick but could we use format `<project-name>-<arch>`? Meaning `bmc-test-go-x86` or `bmc-test-go-arm64` or `bmc-test-go-armv5`?

There are plenty of ARM architectures and they are not always compatible with each other (32b vs 64b, etc).

Reply:
Changed it to be more descriptive. Also added a symlink so we can keep the tool name the same for now.